parser/model/model.go

```diff
@@ -1866,6 +1866,7 @@ const (
 	RunawayActionDryRun RunawayActionType = iota
 	RunawayActionCooldown
 	RunawayActionKill
+	RunawayNoneAction
```

Comment:
NIT: better let this default RunawayNoneAction to be the iota

Reply:
> Seems this default value is useless

Because we do not support patch in the runaway setting clause, it means that the user must declare the configuration of action, but action is just an int and cannot judge nil like a pointer, so this default value can judge whether the user has declared action.

> RunawayNoneAction

Because there is no RunawayNoneAction in kvproto, I have made RunawayNoneAction equal to 3 to avoid semantic conversion between kvproto and tidb

Reply:
> Because we do not support patch in the runaway setting clause, it means that the user must declare the configuration of action, but action is just an int and cannot judge nil like a pointer, so this default value can judge whether the user has declared action.

When action = RunawayNoneAction, we can just set to ResourceGroupSettings and vice verse, so there is even no need to add this action type here.